src/ts/uiutils.ts

```diff
@@ -21,4 +21,133 @@ export namespace UIUtils {
     // Walk and configure the component tree
     recursiveTreeWalker(component);
   }
+
+  // From: https://github.com/nfriend/ts-keycode-enum/blob/master/Key.enum.ts
+  export enum KeyDownKey {
```

Comment:
Do we need all of those? 🤔 Currently, we just care about the arrow keys right?
naming suggestion
```suggestion
  export enum KeyCode {
```